js/schemas/index.ts

```diff
@@ -0,0 +1,2 @@
+import './data';
+import './schemas';
```

Comment:
A. If you just want to export without a default aka to use `import { SCHEMA_INFOS } from '@frequency-chain/schemas'`:
```suggestion
export * from './data';
export * from './schemas';
```

B. If you want to export nested aka to use `import { data } from '@frequency-chain/schemas'`:

```suggestion
export * as data from './data';
export * as schemas from './schemas';
```

C. If you want to export with a default with flat aka to use `import schemas from '@frequency-chain/schemas'`:

```suggestion
import * as data from './data';
import * as schemas from './schemas';

export default { ...data, ...schemas };
```


D. (Suggested?) Export with a default with flat and each function (aka to use A and C)


CJS Tested using:

`npm run build && cd dist && npm pack && cd ..`

And then in a new `npm init` with an `npm i ../../<etc...>`

index.js
```javascript
const schemas = require("@frequency-chain/schemas");

console.log(schemas);
```

Reply:
Thanks for great suggestions. So combining A + C together. One thing I noticed after doing suggested A+C. I have to use `schemas.default.FULL_NAME_TO_ID` to be able to access it. It's wrapped inside the default and I can not access it directly like `schemas.FULL_NAME_TO_ID`

Reply:
Depends on how you do the import at that point. Currently these all work for me (which is ✅ ):

```typescript
import { SCHEMA_INFOS } from "@frequency-chain/schemas";

console.log(SCHEMA_INFOS);
```


```typescript
import * as schemas from "@frequency-chain/schemas";

console.log(schemas.SCHEMA_INFOS);
```


```typescript
import schemas from "@frequency-chain/schemas";

console.log(schemas.SCHEMA_INFOS);
```

```javascript
const { SCHEMA_INFOS } = require("@frequency-chain/schemas");
console.log(SCHEMA_INFOS);
```

```javascript
const schemas = require("@frequency-chain/schemas");
console.log(schemas.SCHEMA_INFOS);
```